src/content/docs/crowdin/online-editor/expression-syntax-elements.mdx

```diff
@@ -4,6 +4,267 @@ description: Learn about the different elements of the expression syntax
 slug: expression-syntax-elements
 ---
 
+import { Aside, LinkCard } from '@astrojs/starlight/components';
 import Include from '~/components/Include.astro';
 
+You can configure verbal expressions and custom placeholders using a shared set of expression syntax elements. These elements define how specific parts of a string should be matched, based on structure, character types, or formatting patterns.
+
+## Expression Elements
+
 <Include file="expression-syntax-elements.mdx" />
+
+## Verbal Expressions
+
+In the Editor, you can use verbal expressions in the [Advanced Filter](/online-editor/#advanced-filter) to search for strings that match specific patterns, similar to regular expressions. You can use it to filter strings based on patterns like punctuation, character types, length, and more.
+
+For example, to find all strings that start with an uppercase letter and end with a period, use the following expression:
+
+```graphql
+start "true", range "A,Z", anything, range "a,z,A,Z,0,9", limit "1,50", then ".", end "true"
+```
+
+This expression means:
+- `start "true"` – the string begins at the start
+- `range "A,Z"` – starts with a capital letter
+- `anything` – followed by any characters
+- `range "a,z,A,Z,0,9"` – allows letters and numbers
+- `limit "1,50"` – string length between 1 and 50 characters
+- `then "."` – ends with a period
+- `end "true"` – end of the string
+
```

Comment:
Add the card grid (two-column) here - with the "Try Verbal Expressions GPT" and "Verbal Tester" cards.

Also, remove the `When configuring verbal expressions, use start "true" to indicate the beginning of the string and end "true" to indicate the end of the string.` tip. It's no longer needed.